lib/improver/utilities/cube_metadata.py

```diff
@@ -413,3 +558,20 @@ def delete_attributes(cube, patterns):
 
     for key in grid_attributes:
         cube.attributes.pop(key)
+
+
+def add_history_attribute(cube, values):
+    """Add a history attribute to a cube. This uses the current datetime to
+    generate the timestamp for the history attribute. The new history attribute
+    will overwrite any existing history attribute.
+
+    Args:
+        cube (iris.cube.Cube):
+            The cube to which the history attribute will be added.
+        values (list):
+            List usually of the form ["add", "Description"] with "description"
+            extra details that are to be included within the history attribute.
```

Comment:
Somewhere in this docstring it wouldn't hurt to say the resulting history attribute has the form "Timestamp: "Description""

Reply:
I've added this information to the docstring of this function.